### PR TITLE
Bugfix/minimalistic checkbox

### PR DESCRIPTION
**Pull request by DeepDiver1975**
Screenshot_2019-03-21 ownCloud Design System(3)

### CODE REVIEW

File: src/elements/OcCheckbox.vue [hunk 1]
```diff
@@ -23,34 +16,39 @@ export default {
...
-</div>
-```
+  ```jsx
+  <div>
+    <section>
+      <h3 class="uk-heading-divider">
+        Checkboxes Types
+      </h3>
+      <oc-checkbox />
+      <oc-checkbox :model="true"/>
```
[the remark this answers is not in the document]
Comment (DeepDiver1975) [reply src/elements/OcCheckbox.vue]: Can you elaborate how this shall look like? THX

[the remark this answers is not in the document]
Comment (felixheidecke) [reply src/elements/OcCheckbox.vue]: `<oc-checkbox v-model="true"/>`

[the remark this answers is not in the document]
Comment (DeepDiver1975) [reply src/elements/OcCheckbox.vue]: please review once more @felixheidecke - THX

**Comment by felixheidecke** [comment]
Very nice

**Pull request merged**